src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -27,7 +27,7 @@ public SchemaClient(IHttpClientFactory httpClientFactory)
 
     public void SetUri(Uri uri)
     {
-        if (_httpClient.BaseAddress == null)
+        if(_httpClient.BaseAddress == null)
         {
             _httpClient.BaseAddress = uri;
```

Comment:
Why is the `BaseAddress` bound so late? Shouldn't we set the `BaseAddress` via DI such that the correct value is prsent by the time it is populated in the ctor? I know you didn't write this initially, do you know if this is possible?

Reply:
Addressed in latest commit(s).